core/domain/state_domain_test.py

```diff
@@ -76,7 +95,12 @@ def test_get_all_html_content_strings(self):
             exploration.init_state.interaction.get_all_html_content_strings())
         self.assertEqual(
             html_list,
-            ['<p>Feedback</p>', '<p>list_of_sets_of_html_strings</p>', '', ''])
+            [
+                '<p>Feedback</p>', '<p>list_of_sets_of_html_strings</p>',
+                '<p>list_of_sets_of_html_strings</p>',
+                '<p>list_of_sets_of_html_strings</p>',
```

Comment:
Do not make all these strings the same. Have separate strings per interaction type, otherwise you can't tell which string is supposed to come from where.

Reply:
done